### PR TITLE
bug/298_round_robin_channel_sel

### PR DESCRIPTION
**Pull request by frbattid**
Fixes the issue https://github.com/telefonicaid/fiware-connectors/issues/298

### CODE REVIEW

File: flume/doc/design/architecture.md [hunk 1]
```diff
@@ -35,10 +35,11 @@ You can simply add more sinks consuming events from the same single channel. Thi
 ### Multiple sinks, multiple channels
```
Comment (fgalan) [inline flume/doc/design/architecture.md]: An entry about this change should be added to CHANGES_NEXT_RELEASE file

Comment (frbattid) [inline flume/doc/design/architecture.md]: Fixed in https://github.com/telefonicaid/fiware-connectors/commit/baebd868b650642fec8557b63778e1b18a5020b3

**Comment by fgalan** [comment]
LGTM

**Comment by frbattid** [comment]
Ready to be merged

**Pull request merged**